UI/SidePanel.cpp

```diff
@@ -1870,9 +1892,14 @@ void SidePanel::PlanetPanel::Refresh() {
     if (m_specials_panel)
         m_specials_panel->Update();
 
-    // set stealth browse text
     ClearBrowseInfoWnd();
 
+    // set planetpanel blockade browse text
+    if (!GetSupplyManager().SystemHasFleetSupply(planet->SystemID(), planet->Owner()) && (!planet->Unowned()))
+        SetBrowseInfoWnd(std::make_shared<TextBrowseWnd>(UserString("PL_BLOCKADED"),
```

Comment:
If I use m_planet_blockaded_graphic->SetBrowseInfoWnd(...), the tooltip doesn't show. Any idea why it doesn't?